framework/web_app/src/stores/ProductStore.ts

```diff
@@ -0,0 +1,160 @@
+import { defineStore } from 'pinia';
+import { Loading } from 'quasar';
+import { IOfferSortByOption, OfferSortByOptions } from 'src/helpers/OfferSortByOptions';
+import { isOfferFavourited } from 'src/helpers/ScrapedProductOfferLogic';
+import { Merchant } from 'src/models/Merchant';
+import type { Product } from 'src/models/Product';
+import type { ScrapedProductOffer } from 'src/models/ScrapedProductOffer';
+import MerchantApiService from 'src/services/api/MerchantApiService';
+import ProductApiService, { SearchByTermQuery } from 'src/services/api/ProductApiService';
+import { computed, reactive, ref } from 'vue';
+
+const merchantApiService = new MerchantApiService();
+const productApiService = new ProductApiService();
+
+// TODO:
+// New Features:
+// - Create a relevance algorithm & set as default sortBy
```

Comment:
all good improvements it looks like, happy to explore either git issues or keep to excel sheets, but if we move to git i'd go through and add everything into git rather than have stuff spread out everywhere

Reply:
Fixed the issue with stockcode coming as numbers out of MAPI & added rest to the dump